examples/application_commands/cog_example.py

```diff
@@ -0,0 +1,32 @@
+from nextcord import (
+    Interaction,
+    message_command,
+    slash_command,
+    user_command,
+    Message,
+    Member,
+)
+from nextcord.ext import commands
+
+
+class ApplicationCommandCog(commands.Cog):  # Subclassing commands.Cog as you always do.
+    def __init__(self, bot: commands.Bot):  # takes in bot for the setup later
+        self.bot = bot  # making it global to this class.
+
+    @slash_command(guild_ids=[...])  # isn't really any different to normal commands.
```

Comment:
Maybe make guild_ids a variable instead?

Reply:
not really needed